.eslintrc.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   rules: {
     'prettier/prettier': 'error',
     'react/no-unescaped-entities': ['error', { forbid: ['>', '}'] }],
+    'react/prop-types': 'off',
```

Comment:
Looks like this rule is enforced with one of the eslint version bumps. This project wasn't using prop types before so this rule is disabled for now.